lib/btrfs_test.pm

```diff
@@ -57,7 +85,33 @@ move from rescue to default target, logs us out. Die if DBus is active at this p
 it means that DBus got activated somehow, thus invalidated `snapper --no-dbus` testing.
 =cut
 sub snapper_nodbus_restore {
-    if (script_run('systemctl is-active dbus')) {
+
+    my $rescue_service_config_path = "/usr/lib/systemd/system/rescue.service";
+
+    type_string "tty";
+    wait_serial "tty";
+    type_string "\n";
+    if (wait_serial('\/dev\/hvc\d+', 5)) {
```

Comment:
Is there a specific reason why this must appear within 5 seconds? Better use the default timeout unless you want to catch something that is really time critical.

Reply:
There is no reason for it to having to appear within 5 seconds. I did that because it appeared a huge delay for simple serial console I/O.
But I will remove it because it's not necessary and might also look confusing.